react/Hidden/Hidden.js

```diff
@@ -5,21 +5,30 @@ import classNames from 'classnames';
 
 import styles from './Hidden.less';
 
-const Hidden = ({ children, className, print, screen, mobile, desktop }) => (
-  <span
-    className={classNames(
-      className,
-      {
+const Hidden = ({ children, component, className, print, screen, mobile, desktop, ...restprops }) => {
+  const Component = component ? component : 'span';
```

Comment:
Isn't this line redundant with the `defaultProps` defined below?

Reply:
Totally, will remove

Reply:
Done 👍